### ISSUE TITLE
Add extracted keywords to each summary level to further human guidance

### ISSUE THREAD

**Issue by headchem** [headchem/StoryGhostPlotter#105]
Similar to how we used KeyBERT (https://colab.research.google.com/drive/1gtDaXTg5E17aXfrqv-899KRfTUoH_W2P?authuser=2#scrollTo=K9snOYJ395hy) to extract keywords from loglines, we could do the same for blurbs, expanded, scene summary, and scene full. Then we could train two independent sets of models, one with keywords as part of the prompt, and one without. Then the user can choose whether they want the next blurb/expanded/scene summary/full to include certain keywords, or if left blank, it'll be the keyword-less models we've already trained. Could also boost the logits of specific character names to encourage the model to generate events involving the specified characters. This could be done in conjunction with #103
